### DISCUSSION TITLE
Tamagotchi/Virtual pet

### DISCUSSION THREAD

**Discussion by ThatOneCalculator** [InfiniTimeOrg/InfiniTime#1524] [Ideas]
### Verification

- [X] I searched for similar feature request and found none was relevant.

### Pitch us your idea!

A small tamagotchi-like app 

### Description

It might be a fun app for people nostalgic for Tamagotchi and other virtual pets, perhaps one could keep it happy/healthy with steps

**Comment by Danimations** [comment]
I would love to design some creatures and animations for this if the idea gets some traction.

**Comment by 53c70r** [comment]
Bluetooth could be used to socialize your creature and pair them.

**Comment by coon58** [comment]
That would be really cool! Great idea!

**Comment by ghost** [comment]
With the external resource loading in 1.11, we could also perhaps use external images as "pet" sprites.

**Reply by Danimations** [reply]
Sounds good to me. I'd love to make some pixel art critters for this. Low-resolution with a little animation would suit the retro throwback feeling. Could someone who's spent more time with tamagochi than I have write up a little brief, describing the basic functions and therefore scoping what art/animations would be required?

**Reply by ThatOneCalculator** [reply]
Generally you're going to want

- Idle animation
- Turn around
- Up close
- Eating
- Playing
- Dying

For all of the stages (usually 3) for each character. 

That's just the very basics, lots of tamagotchi and tamagotchi-likes add more.

**Comment by ajira86** [comment]
I'm working of something like but this is only an experiment at this time
